### PR TITLE
[MERGE ON JAN 22] Update Analytics and Audience Docs for 2.0

### PR DESCRIPTION
**Pull request by nporter-adbe**
- Updates the import statements for static libraries
- Adds documentation for a new API in Analytics for customer visitor ID

### CODE REVIEW

**Comment by nporter-adbe** [comment]
@rekrishn Unable to tag you as a reviewer.

File: using-mobile-extensions/adobe-analytics/analytics-event-reference.md [hunk 1]
```diff
@@ -158,16 +158,16 @@ Here is an example of event data for Track request
...
 
-This event is responsible for fetching the Analytics tracking identifier \(AID\) from the Analytics extension.
+This event is responsible for fetching the Analytics tracking identifier \(AID\) and custom visitor identifier (VID) from the Analytics extension.
 
 This event will be generated the `analyticsGetTrackingIdentifier` public API call.
 
 On receiving the analytics identity request event, Analytics module completes one of the following tasks:
 
-* Checks the persistence for the saved aid.
+* Checks the persistence for the saved aid and vid.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/analytics-event-reference.md]: Please capitalize aid and vid to AID and VID as they are acronyms.

File: using-mobile-extensions/adobe-analytics/analytics-event-reference.md [hunk 2]
```diff
@@ -158,16 +158,16 @@ Here is an example of event data for Track request
...
 * If `nil`, creates a sync network request to the analytic server to retrieve aid.
 
-Dispatches an Analytics Identity Response Event with the aid value
+Dispatches an Analytics Identity Response Event with the aid and vid value.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/analytics-event-reference.md]: Please capitalize "aid" and "vid"

**Comment by nporter-adbe** [comment]
@rekrishn Thanks for the feedback :). I'll come back and merge this on Tuesday when we release.

**Pull request merged**